spec/factories/invoice_items.rb

```diff
@@ -1,6 +1,10 @@
+require "faker"
+
 FactoryBot.define do
   factory :invoice_item do
     quantity { rand(1..10) }
+    unit_price { Faker::Number.number(digits: 4) }
```

Comment:
I wasn't too 100% sure or not. Are item unit_price and the unit_price on a related item going to be the same value?

It doesn't fully matter I think in the factory, but we could tie these to be the same value if they are the same.

Reply:
Good point. If factory bot is making the objects the current config will produce different random numbers. The reality is that field should be a computed property where unit_price is the product of item price and the quantity. And not stored in the DB as a rote number